### PR TITLE
0037 remove profile limit

### LINKED ISSUE #37: Remove profile limit
Remove the limit to the number of profiles that can be added to the application. If the profile limit is maxed out to 256 profiles, please include an error message if a user exceeds that limit.

### PR DESCRIPTION
**Pull request by DSMatusevich**
**Description**:
- Increased the profile limit to 32
- Addressed a problem with long names in the Home pane
- Increased version number

**Related issue(s)**:

Fixes #37 

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: tools-core/src/test/java/com/hedera/hashgraph/client/core/helpers/CollatorHelperTest.java [hunk 1]
```diff
@@ -26,6 +26,7 @@
 import com.hedera.hashgraph.client.core.transactions.SignaturePair;
 import com.hedera.hashgraph.sdk.AccountId;
 import com.hedera.hashgraph.sdk.AccountInfo;
+import com.hedera.hashgraph.sdk.PublicKey;
```
Comment (qnswirlds) [inline tools-core/src/test/java/com/hedera/hashgraph/client/core/helpers/CollatorHelperTest.java:29]: Should remove this unused import

File: tools-ui/src/main/java/com/hedera/hashgraph/client/ui/CreatePaneController.java [hunk 2]
```diff
@@ -94,6 +94,7 @@
 import org.apache.logging.log4j.Logger;
 import org.bouncycastle.util.encoders.Hex;
 import org.controlsfx.control.ToggleSwitch;
+import org.jetbrains.annotations.NotNull;
```
Comment (qnswirlds) [inline tools-ui/src/main/java/com/hedera/hashgraph/client/ui/CreatePaneController.java:97]: Should remove this unused import

**Review by qnswirlds — APPROVED** [review]
just nits.

**Review by qnswirlds — APPROVED** [review]

**Review by MarcKriguerAtHedera — APPROVED** [review]
All changes (now that the unused imports are gone) look good to me.

**Pull request merged**